### PR TITLE
Add failing spec for STI associations issue (#47)

### PR DESCRIPTION
**Pull request by pzac**

### CODE REVIEW

**Comment by vjt** [comment]
Thank you @pzac. This is super helpful.

File: spec/support/helpers.rb [hunk 1]
```diff
@@ -158,6 +162,12 @@ class ::Plain < ActiveRecord::Base
         class ::Event < ActiveRecord::Base
           extend ChronoModel::TimeMachine::TimeQuery
         end
+
+        class ::Animal < ActiveRecord::Base
+          include ChronoModel::TimeMachine
+        end
+
+        class ::Dog < Animal; end
```
Comment (lleirborras) [inline spec/support/helpers.rb]: I was expecting a 🐐 somewhere and instead I got a 🐕

Comment (vjt) [reply spec/support/helpers.rb]: TROLOLOL

ref: https://www.youtube.com/watch?v=2Z4m4lnjxkY

Comment (vjt) [reply spec/support/helpers.rb]: You have your Goat now: [🐐](https://github.com/ifad/chronomodel/pull/86/files#diff-b1e066867ad264cea50a52d721ed0d49R18)

**Pull request merged**